antarest/study/business/allocation_management.py

```diff
@@ -69,15 +69,17 @@ class AllocationMatrix(FormFieldsBaseModel):
     data: 2D-array matrix of consumption coefficients
     """
 
+    model_config = ConfigDict(arbitrary_types_allowed=True)
+
     index: Annotated[List[str], Len(min_length=1)]
     columns: Annotated[List[str], Len(min_length=1)]
-    data: List[List[float]]  # NonNegativeFloat not necessary
+    data: npt.NDArray[np.float64]
```

Comment:
Actually I think we could re-use the `NpArray` annotated type which was introduced in `antarest.study.model`.

We should probably move it to a common file, for example in `core.serde`